client/src/app/pages/license-list/license-table.tsx

```diff
@@ -0,0 +1,163 @@
+                      })}
+                    >
+                      <WithPackagesByLicense
+                        key={item.license}
+                        licenseId={item.license}
+                      >
+                        {(totalPackages, isFetching, fetchError) => (
+                          <LoadingWrapper
+                            isFetching={isFetching}
+                            fetchError={fetchError}
+                            isFetchingState={
+                              <Skeleton screenreaderText="Loading contents" />
+                            }
+                            fetchErrorState={(error) => (
+                              <TableCellError error={error} />
+                            )}
+                          >
+                            {totalPackages && totalPackages > 0 ? (
+                              <NavLink
+                                to={generatePath(Paths.packagesFilter, {
+                                  licenseName: item.license,
+                                })}
+                              >
+                                {`${totalPackages} Package${totalPackages > 1 ? "s" : ""}`}
+                              </NavLink>
+                            ) : (
+                              "0 Packages"
+                            )}
+                          </LoadingWrapper>
+                        )}
+                      </WithPackagesByLicense>
+                    </Td>
+                    <Td
+                      width={50}
+                      {...getTdProps({
+                        columnKey: "sboms",
+                        item: item,
+                        rowIndex,
+                      })}
+                    >
+                      <WithSBOMsByLicense
+                        key={item.license}
+                        licenseId={item.license}
+                      >
+                        {(totalSBOMs, isFetching, fetchError) => (
+                          <LoadingWrapper
+                            isFetching={isFetching}
+                            fetchError={fetchError}
+                            isFetchingState={
+                              <Skeleton screenreaderText="Loading contents" />
+                            }
+                            fetchErrorState={(error) => (
+                              <TableCellError error={error} />
+                            )}
+                          >
+                            {totalSBOMs && totalSBOMs > 0 ? (
+                              <NavLink
+                                to={generatePath(Paths.packagesFilter, {
+                                  licenseName: item.license,
+                                })}
+                              >
+                                {`${totalSBOMs} Package${totalSBOMs > 1 ? "s" : ""}`}
+                              </NavLink>
+                            ) : (
+                              "0 Packages"
```

Comment:
Same as above 
see that the 2 columns are rendering the same Word 

<img width="340" height="105" alt="Image" src="https://github.com/user-attachments/assets/2639e653-cfbe-43f1-bfa8-6d1da87041e9" />